news/_posts/core-weekly/2018-12-20-coreweekly-week-50-2018.md

```diff
@@ -0,0 +1,207 @@
+
+
+## A quick update about GitHub issues
+
+Last week, [37 new issues](https://github.com/PrestaShop/PrestaShop/issues?utf8=%E2%9C%93&q=is:issue+created:2018-12-10..2018-12-16) have been created in PrestaShop's core repository, and [8 fixed issues](https://github.com/PrestaShop/PrestaShop/issues?utf8=%E2%9C%93&q=is:issue+label:fixed+closed:2018-12-10..2018-12-16) have been closed.
+
+## Code changes in the 'develop' branch (for v1.7.6.0)
+
+### Core
+
+* [#10533](https://github.com/PrestaShop/PrestaShop/pull/10533): CO: add old product ID on actionProductAdd hook when we duplicate product. Thank you [@duGuillaume](https://github.com/duGuillaume)
+* [#10975](https://github.com/PrestaShop/PrestaShop/pull/10975): Introduced a new function for Products.php for retrive a product ID given it reference. Thank you [@runningz](https://github.com/runningz)
+* [#11635](https://github.com/PrestaShop/PrestaShop/pull/11635): improve use of protocol. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11643](https://github.com/PrestaShop/PrestaShop/pull/11643): Additional PHP Improvements. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11645](https://github.com/PrestaShop/PrestaShop/pull/11645): Fix discount computing to take combination into account, by [@jolelievre](https://github.com/jolelievre)
+* [#11653](https://github.com/PrestaShop/PrestaShop/pull/11653): Allowed the override of  _PS_DEBUG_PROFILING_. Thank you [@SebBareyre](https://github.com/SebBareyre)
+* [#11666](https://github.com/PrestaShop/PrestaShop/pull/11666): Fully qualified strict types. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11667](https://github.com/PrestaShop/PrestaShop/pull/11667): Heredoc to nowdoc. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11669](https://github.com/PrestaShop/PrestaShop/pull/11669): Improve error suppression. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11671](https://github.com/PrestaShop/PrestaShop/pull/11671): Use constant instead of function. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11672](https://github.com/PrestaShop/PrestaShop/pull/11672): Non printable character. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11678](https://github.com/PrestaShop/PrestaShop/pull/11678): Fix use of protocol. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11679](https://github.com/PrestaShop/PrestaShop/pull/11679): Fix issue with deleting a category with subcategories on Multistore. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11685](https://github.com/PrestaShop/PrestaShop/pull/11685): php-cs-fixer: Fix "non printable character". Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#11689](https://github.com/PrestaShop/PrestaShop/pull/11689): Merge 1.7.5.x in develop, by [@PierreRambaud](https://github.com/PierreRambaud)
```

Comment:
Dont know if this one is relevant as it is a "util" Pull Request used to backport fixes on 1.7.5.x branch onto develop

Reply:
I usually include it, this is the first time someone notice it ;-)
let's keep it.